Framework/Muon/inc/MantidMuon/MuonAlgorithmHelper.h

```diff
@@ -0,0 +1,78 @@
+#ifndef MANTID_MUON_MUONALGORITHMHELPER_H_
+#define MANTID_MUON_MUONALGORITHMHELPER_H_
+
+#include "MantidAPI/GroupingLoader.h"
+#include "MantidAPI/MatrixWorkspace.h"
+#include "MantidAPI/WorkspaceGroup.h"
+#include "MantidKernel/System.h"
+
+#include <string>
+#include <vector>
+
+namespace Mantid {
+namespace Muon {
+/// Types of entities we are dealing with
+const enum ItemType { Pair, Group };
```

Comment:
A name for these types is great. C++11 introduced the concept of ["Scoped Enumerations](http://en.cppreference.com/w/cpp/language/enum) that makes them even better and these should generally be preferred these days. Can we use these here?